docs/src/main/asciidoc/building-native-image-guide.adoc

```diff
@@ -262,6 +267,53 @@ docker run -i --rm -p 8080:8080 quarkus-quickstart/getting-started
 
 NOTE: Interested by tiny Docker images, check the {quarkus-images-url}/graalvm-{graalvm-version}/distroless[distroless] version.
 
+== Producing a container with a multi-stage Docker build
+
+On the previous paragraph, we show you how to build a native image with Maven, however this implies that GraalVM is installed locally, and if you use
+a CI/CD pipeline, inside your CI/CD.
+
+If not, you can use Docker to make the Maven build by using a multi-stage Docker build. A multi-stage Docker build is like two Dockerfile files combined in one, the first is used to build the artifact used by the second. 
```

Comment:
`If not` -> `In cases where the GraalVM requirement cannot be met`

Reply:
done